.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
```

Comment:
- [ ] This is the first project in which you should set up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository. I can see that you have added the eslintrc file great job 👏🏻  but you have a problem with your linter configuration because it doesn't run, please let's follow [these instructions](https://github.com/microverseinc/linters-config/tree/master/javascript) on how to setup the linters correctly.